### PR TITLE
[3340] Add start page

### PR DESCRIPTION
**Pull request by dankmitchell**
### After

https://s121d02-3340-ptt-as.azurewebsites.net/

localhost_3000_(Laptop with MDPI screen)

### Checklist
- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   get "/auth/dfe/signout", to: "sessions#destroy"
   get "/auth/failure", to: "sessions#failure"
 
-  root to: "providers#index"
+  root to: "pages#start"
```
Comment (misaka) [inline config/routes.rb:17]: Should this be optional?

Comment (misaka) [reply config/routes.rb:17]: i.e. can we switch this based on `Settings.show_start_page`, so we can turn it off without redeploying.

Comment (dankmitchell) [reply config/routes.rb:17]: MVP

Comment (misaka) [reply config/routes.rb:17]: :shipit:

Comment (misaka) [reply config/routes.rb:17]: Happy to approve once you're happy with it to come out of draft.

**Comment by aliuk2012** [comment]
Closed because #1061 implements these changes

**Pull request closed without merging**